ConsolePort/Libs/ActionButton.lua

```diff
@@ -1367,8 +1375,7 @@ Action.GetSpellId              = function(self)
 	if actionType == 'spell' then
 		return id
 	elseif actionType == 'macro' then
-		local _, _, spellId = GetMacroSpell(id)
-		return spellId
+		return GetMacroSpell(id)
```

Comment:
@seblindfors `GetMacroSpell` returns only one value. If you don't want to take the whole PR then this is actually a standalone fix for glow on macros